libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnAssignEntity.uischema

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema",
+    "form": {
+        "order": [
+            "condition",
+            "*"
+        ],
+        "hidden": [
+            "actions"
+        ],
```

Comment:
Does this mean hidden on the right property pane and only in the design surface?

Reply:
Correct